Dockerfile

```diff
@@ -23,7 +23,7 @@ WORKDIR /app
 COPY ui/package.json .
 COPY ui/pnpm-lock.yaml .
 
-RUN pnpm install
+RUN pnpm install --frozen-lockfile
```

Comment:
The problem was in the Dockerfile. It does `pnpm install`  as the first step so it will already modify the lockfile and when the checks are running a bit later it will not detect any further changes. [Now it fails as it should](https://github.com/direktiv/direktiv/actions/runs/12235432866/job/34126820601?pr=1728).